src/olympia/discovery/data.py

```diff
@@ -24,46 +24,48 @@ def __repr__(self):
 # are used instead of slugs to prevent any accidental replacement of a deleted
 # add-on by another.
 discopane_items = [
-    # 'All Fall' theme.
-    DiscoItem(addon_id=44686, type=amo.ADDON_PERSONA, addon_name='All Fall'),
+    # 'Owl First Snow' theme.
+    DiscoItem(
+        addon_id=676070, type=amo.ADDON_PERSONA, addon_name='Owl First Snow'),
 
-    # Search Image
+    # uBlock Origin
     DiscoItem(
-        addon_id=455926,
-        addon_name='Search Image',
-        heading=_(u'Easily find images {start_sub_heading}with {addon_name}'
+        addon_id=607454,
+        heading=_(u'Block ads {start_sub_heading}with {addon_name}'
                   u'{end_sub_heading}'),
         description=string_concat(
             '<blockquote>',
-            _(u'Highlight any text and right-click to quickly perform an '
-              u'image search.'),
+            _(u'A lightweight and effective ad blocker. uBlock Origin '
+              u'enforces thousands of content filters without chewing up a '
+              u'bunch of memory.'),
             '</blockquote>')),
 
-    # Enhancer for YouTube
+    # LastPass
     DiscoItem(
-        addon_id=700308,
-        heading=_(u'Improve videos {start_sub_heading}with {addon_name}'
+        addon_id=8542,
+        addon_name='LastPass',
+        heading=_(u'Manage passwords {start_sub_heading}with {addon_name}'
                   u'{end_sub_heading}'),
         description=string_concat(
             '<blockquote>',
-            _(u'Enjoy a suite of new YouTube features, like cinema mode, '
-              u'ad blocking, auto-play control, and more.'),
+            _(u'Simplify and sync all your various website logins across '
+              u'devices with one password to rule them all.'),
             '</blockquote>')),
 
-    # 'Snub Nosed Dog' theme.
+    # 'Tiffy01' theme (slug = color-to-color).
     DiscoItem(
-        addon_id=376685, type=amo.ADDON_PERSONA, addon_name='Snub Nosed Dog'),
+        addon_id=290486,
+        type=amo.ADDON_PERSONA),
```

Comment:
Does this need `addon_name`?

Reply:
The name in production looked alright so I'm not overwriting it.